src/components/Table/TasksTable.tsx

```diff
@@ -38,6 +38,7 @@ interface TasksTableProps {
   onExpand?: (expanded: boolean) => void;
   sorts?: SortOrder[];
   statusSelectorProps?: TreeSelectProps;
+  onColumnHeaderClick?: (sortField) => void;
```

Comment:
Should this be a required prop?  I'm seeing this error in [storybook](https://mciuploads.s3.amazonaws.com/spruce/spruce_ubuntu1604_storybook_patch_573bf1240c6a1ad736f6a5bad1dbbcec1a718177_617190292a60ed33341e467f_21_10_21_16_07_06/storybook/index.html?path=/story/tasks-table--base-task-table)
![image](https://user-images.githubusercontent.com/4605522/138357194-84919951-4b6a-4da4-afbf-0cd509a57208.png)

You might need to add it to the storybook story.